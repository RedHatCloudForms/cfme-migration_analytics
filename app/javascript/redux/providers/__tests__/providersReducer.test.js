// TODO add unit tests for providersReducer.js

it('works', () => expect(true).toBe(true));
