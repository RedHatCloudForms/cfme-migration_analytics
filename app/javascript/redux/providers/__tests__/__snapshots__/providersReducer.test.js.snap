// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`providers reducer constructs initial state correctly 1`] = `
Object {
  "errorFetchingProviders": null,
  "isFetchingProviders": false,
  "providers": null,
}
`;
